toolium/behave/environment.py

```diff
@@ -72,10 +80,9 @@ def before_feature(context, feature):
         no_driver = 'no_driver' in feature.tags
         start_driver(context, no_driver)
 
-    # Dictionary to store information between steps
-    context.storage = dict()
     # Dictionary to store information between features
     context.feature_storage = dict()
+    context.storage = collections.ChainMap(dict(), context.feature_storage, context.run_storage)
```

Comment:
```suggestion
    context.storage = collections.ChainMap(context.feature_storage, context.run_storage)
```

Reply:
Both ways seems to work correctly. So choose your preference.